hw_diag/tests/diagnostics/test_env_var_diagnostics.py

```diff
@@ -56,5 +56,7 @@ def test_env_vars_success(self):
             'FIRMWARE_VERSION': 'foo',
             'FW': 'foo',
             'VARIANT': 'foo',
-            'VA': 'foo'
+            'VA': 'foo',
+            'FIRMWARE_SHORT_HASH': 'foo',
+            'firmware_short_hash': 'foo'
```

Comment:
Why do we have both lowercase and upper case here?

Reply:
Good question. I've kept the existing format before and of course the last change made it look pretty ugly (only case change). Should I remove the upper case from here?

Reply:
Before that it was only `FH` for backward compatibility.

Reply:
Added the comment as we talked in the meeting.